fuzzers/klee/fuzzer.py

```diff
@@ -24,9 +24,56 @@
 from fuzzers.afl import fuzzer as afl_fuzzer
 
 
+def is_benchmark(name):
+    """ Check if the benchmark contains the string 'name' """
+    # TODO: benchmark = os.getenv("BENCHMARK", None)
+    # return benchmark is not None and name in benchmark
+
+    # OSS-fuzz benchmarks do not have a consistent folder content
+    # but typically have the name of the benchmark in the PWD
+    pwd_dir = os.getenv("PWD", None)
+    print(pwd_dir)
+    if pwd_dir is None:
```

Comment:
Btw instead of doing os.getenv(..., None) and excepting if None, you can do os.environ[...] which excepts if it isn't set.